### PR TITLE
feat: added the `postprocessor` option

### PR DESCRIPTION
**Pull request by alexander-akait**
This PR contains a:

- [ ] **bugfix**
- [x] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

fixes https://github.com/webpack-contrib/html-loader/issues/291

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ logs
 *.log
 npm-debug.log*
 .eslintcache
+.cspellcache
```
Comment (evenstensberg) [inline .gitignore:11]: We can also disable cache in cpsell. It doesn't differ much from no-cache

File: test/postprocessor-option.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      "module",
+    );
+    expect(
+      execute(readAsset("main.bundle.js", compiler, stats)),
+    ).toMatchSnapshot("result");
+    expect(getWarnings(stats)).toMatchSnapshot("warnings");
+    expect(getErrors(stats)).toMatchSnapshot("errors");
+  });
+
+  it('should work with Async "postprocessor" Function option', async () => {
```
Comment (evenstensberg) [inline test/postprocessor-option.test.js:33]:
```suggestion
  it('should work with async "postprocessor" function option', async () => {
```

File: .gitignore (hunk 1 above)
Comment (alexander-akait) [reply .gitignore:11]: It has a big changes when you have a lot of files, I don't think we need disable it, we use cache everywhere

Comment (evenstensberg) [reply .gitignore:11]: ok

**Pull request merged**